test/explorer/data_frame_test.exs

```diff
@@ -691,6 +691,30 @@ defmodule Explorer.DataFrameTest do
              }
     end
 
+    test "with map " do
+      df = DF.new(%{a: [1, nil, 3], b: ["a", "b", nil]})
+      df1 = DF.mutate(df, c: %{a: a, b: b, lit: 1, null: is_nil(a)})
```

Comment:
We should also add a column which is a regular series. We actually supported this originally in the first commit but it was lost in the refactor.

Reply:
Something like this ?
```
s = Series.from_list(["x", "y", "z"])
DF.mutate(df, c: s)
```

Reply:
Yes!

Reply:
Can i look at this sometime later this week ?

Reply:
There is no rush! We can wait and merge it once you have time to look at it. :)